evaluation/views.py

```diff
@@ -127,6 +131,9 @@ def submission(request, sub_id=None):
 def compare(request, sub_a_id=None, sub_b_id=None):
     """Renders vertical or horizontal comparison between two submissions."""
 
+    if request.method == "POST" and sub_a_id is None and sub_b_id is None:
```

Comment:
@snukky: what's the rationale? Don't you need the submission IDs for storing a comparison result?